File: test_preview_files/sample.py
#!/usr/bin/env python3
"""Sample Python file for preview testing."""

def hello_world():
    """Print hello world message."""
    print("Hello, World!")
    return True

class Calculator:
    """A simple calculator class."""
    
    def __init__(self):
        self.result = 0
    
    def add(self, a, b):
        """Add two numbers."""
        self.result = a + b
        return self.result
    
    def subtract(self, a, b):
        """Subtract two numbers."""
        self.result = a - b
        return self.result

if __name__ == "__main__":
    hello_world()
    calc = Calculator()
    print(f"5 + 3 = {calc.add(5, 3)}")

File: test_preview_files/Sample.cs
using System;

namespace PreviewTest
{
    /// <summary>
    /// Sample C# class for preview testing.
    /// </summary>
    public class Sample
    {
        public string Name { get; set; }
        public int Value { get; set; }
        
        public Sample(string name, int value)
        {
            Name = name;
            Value = value;
        }
        
        public void PrintInfo()
        {
            Console.WriteLine($"Name: {Name}, Value: {Value}");
        }
    }
    
    class Program
    {
        static void Main(string[] args)
        {
            var sample = new Sample("Test", 42);
            sample.PrintInfo();
        }
    }
}
